test/functional/commands/recordscreen-e2e-specs.js

```diff
@@ -42,7 +42,7 @@ describe('recording the screen', () => {
         data = await driver.adb.shell(cmd); 
       }
       
-      isEmulated = await driver.adb.isEmulatorConnected('emulator-5554');
+      isEmulated = driver.adb.isEmulatorConnected('emulator-5554');
```

Comment:
as @imurchie commented, this need to use *await* since _isEmulatorConnected* method is async. Otherwise you would be getting a Promise instead of the actual boolean result.